### PR TITLE
Feature/bug fixes

### PR DESCRIPTION
**Pull request by aleciafb**
Changes
- Fix JS issue where filter items disappear on desktop when clicking on page
- Fix hero text positioning
- Remove time from articles
- Adjust carousel sizes
- Fix issue where quotation mark drops to next line
- Quotation mark styling tweaks
- Render link for media text

### CODE REVIEW

File: girleffect/static_src/site/javascript/main.js [hunk 1]
```diff
@@ -401,9 +401,12 @@ $(function() {
             $('form').submit();
         });
 
+        // NOTE Close select box if clicked anywhere else on page
         $(document).click(function() {
```
Comment (jwfwessels) [inline girleffect/static_src/site/javascript/main.js]: nit: Excuse my lack of context, but usually when one wants to deactivate something (A) when the user interacts with something else you do a check inside the click handler to check if the click propagated to the root element of the of A, else deactivate. just asking cause I don't see any check happening.

File: girleffect/static_src/site/patterns/01-molecules/hero/_hero.scss [hunk 2]
```diff
@@ -57,11 +57,11 @@
             justify-content: center;
             flex-direction: column;
             position: absolute;
-            left: 50%;
```
Comment (jwfwessels) [inline girleffect/static_src/site/patterns/01-molecules/hero/_hero.scss]: nit: This absolute positioning to center something shouldn't be necessary if you can use flexbox (which I see you are using. Consider switching to using `flex` with `justify-content` and `align-items` both set to center on the parent of this element.

Comment (aleciafb) [reply girleffect/static_src/site/patterns/01-molecules/hero/_hero.scss]: @jwfwessels Yes, this was removed. Currently using
```align-items: center;
     justify-content: center;
```

We are only positioning something absolutely over the hero image

File: girleffect/static_src/site/javascript/main.js (hunk 1 above)
Comment (aleciafb) [reply girleffect/static_src/site/javascript/main.js]: @jwfwessels You are right. I'll send you a follow up PR later today for this

**Pull request merged**